delfin/api/v1/alerts.py

```diff
@@ -67,6 +69,11 @@ def show(self, req, id):
         for alert in alert_list:
             alert_util.fill_storage_attributes(alert, storage)
 
+        try:
+            self.alert_exporter.dispatch(ctx, alert_list)
```

Comment:
Any reason for adding alert exporter dispatch in wsgi server? Can we handle it in task manager server?

Reply:
Removed this. there sync API in alert to handle alert pull task from task manager. GET Alert will respond directly to client through REST.